examples/active_support_helper.rb

```diff
@@ -0,0 +1,30 @@
+module ActiveRecord
+  class Base
+  end
+end
+
+module ActiveSupport
+  class TestCase < Minitest::Test
+    def setup(*args, &block)
+    end
+
+    # Remove describe method if present
+    class << self
+      remove_method :describe
+    end if self.respond_to?(:describe) &&
+        self.method(:describe).owner == ActiveSupport::TestCase
+
+    # Add spec DSL
+    extend ::Minitest::Spec::DSL
+
+    if defined?(ActiveRecord::Base)
```

Comment:
`ActiveRecord::Base` is defined 19 lines above this, so won't this if statement always return true?